notes_app/client/app/notes/notes_service.js

```diff
@@ -5,14 +5,22 @@
 
   function notesService($http, $log) {
     var api_path = 'http://localhost:8000/api/v1/note/';
+    var data = {
+      notes: []
+    };
+    function updateList(list){
+      data.notes = list;
+    }
     return {
+      'data': data,
       'list': function(limit, offset) {
         return $http.get(api_path, {
           'params': {
             'limit': limit,
             'offset': offset
           }
         }).then(function successHandler(response) {
+          updateList(response.data.objects);
           return response.data;
```

Comment:
no. here I pass an array of Notes ( ```response.data.objects``` ) to ```updateList()``` function so it sets this arrray to ```data.notes``` , thus ```data.notes``` gets updated